packages/flex/src/lib/Flex/context.ts

```diff
@@ -1,10 +1,14 @@
 import type { CurrentWritable } from '@threlte/core'
-import mitt, { type Emitter } from 'mitt'
+import { type Emitter } from 'mitt'
```

Comment:
```suggestion
import type { Emitter } from 'mitt'
```
...since the above will compile to `import {} from 'mitt'`